go.mod

```diff
@@ -31,9 +31,9 @@ require (
 	github.com/libp2p/go-libp2p-transport-upgrader v0.3.0
 	github.com/libp2p/go-tcp-transport v0.2.1
 	github.com/multiformats/go-multiaddr v0.3.1
-	github.com/onflow/cadence v0.10.1
+	github.com/onflow/cadence v0.11.1-0.20201112191337-3ab62fe97454
```

Comment:
Maybe add a comment here noting why this dependency is pinning an unreleased/untagged version

Reply:
👍